roles/nvidia-gpu-operator/defaults/main.yml

```diff
@@ -19,7 +19,17 @@ gpu_operator_namespace: "gpu-operator-resources"
 gpu_operator_default_runtime: "docker"
 gpu_operator_driver_registry: "nvcr.io/nvidia"
 gpu_operator_driver_version: "460.32.03"
-gpu_operator_plugin_args: "--mig-strategy={{ k8s_gpu_mig_strategy }},--pass-device-specs=true,--fail-on-init-error=true,--device-list-strategy=envvar,--nvidia-driver-root=/run/nvidia/driver"
+deepops_dir: "/opt/deepops"
+gpu_operator_config_dir: "{{ deepops_dir }}/gpu-operator"
+gpu_operator_plugin_args:
```

Comment:
Have you tested that this works? It's used in the below way, and I was having a very hard time getting Ansible to parse all this correctly.

I was trying to avoid checking a new config yml into the repo or creating a temporary one, but it might be about time to do that.

"--set devicePlugin.args={"{{ gpu_operator_plugin_args }}"}"